samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm

```diff
@@ -42,7 +42,6 @@ - (void)viewDidLoad {
         [self setupAccelerometer];
     } else {
         accLabel = [[UILabel alloc] initWithFrame:CGRectMake(30, accHeight, 300, 50)];
-        [accLabel setBackgroundColor:nil];
```

Comment:
In which colour does nil result? Is it different in iOS versus WinObjC? Should it ever be clear?